log/log.go

```diff
@@ -126,6 +127,9 @@ func (w *bothWriter) logAtLevel(level syslog.Priority, msg string) {
 	const red = "\033[31m\033[1m"
 	const yellow = "\033[33m"
 
+	crc := crc32.ChecksumIEEE([]byte(msg))
+	msg = fmt.Sprintf("%x %s", crc, msg)
```

Comment:
This is borderline bike-shedding but I think given the CRC is variable length it might make parsing it easier to have some explicit delimiters above and beyond the space separated style. E.g.:

```
fmt.Sprintf("[%x] %s", crc, msg)
```

WDYT?

Reply:
/shrug, works for me, but I don't really have strong opinions.

Reply:
I don't feel strongly either. I +1'd this branch as-is and we can see whether @jsha has thoughts on his :mag: round.

Reply:
agreed I'd like to keep this constant-length so log lines line up cleanly (until they don't). To save 2 bytes let's use base64.RawURLEncoding. To avoid having to marshal the uint32 into bytes we can use the `Hash32` form of `crc32`.